人工智能/人工智能数学基础/人智数学基础实验/2026春实验一/实验一代码/DFT_best.py

```
@@ -0,0 +1,83 @@
+
+            # 只保留相位 (幅度设为 1)
+            complex_phase_only = 1 * np.exp(1j * phase_block)
+            f_ishift_phase = np.fft.ifftshift(complex_phase_only)
+            recon_phase_only[i:i+8, j:j+8] = np.abs(np.fft.ifft2(f_ishift_phase))
+
+    #可视化
+    plt.rcParams['font.sans-serif'] = ['SimHei'] # 用于正常显示中文
+    plt.rcParams['axes.unicode_minus'] = False
+    
+    fig, axes = plt.subplots(2, 3, figsize=(15, 10))
+    axes[0, 0].imshow(Y_padded, cmap='gray'); axes[0, 0].set_title('原始 Y 分量 (补零后)')
```

Ruff (0.15.17)

[error] Multiple statements on one line (semicolon)

(E702)